src/lsg/patterns/property-items/string-item/index.tsx

```diff
@@ -60,16 +57,14 @@ export const StringItem: React.StatelessComponent<StringItemProps> = props => {
 
 	return (
 		<StyledStringItem className={className}>
-			<label>
-				<StyledLabel>{label}</StyledLabel>
-				<StyledInput
-					onChange={onChange}
-					onBlur={onBlur}
-					type="textarea"
-					value={value}
-					placeholder="Type in"
-				/>
-			</label>
+			<PropertyLabel label={label} />
+			<StyledInput
+				onChange={onChange}
+				onBlur={onBlur}
+				type="textarea"
```

Comment:
I think this should be `"text"`. If you want to use a `<textarea/>` here we'll have to create a distinct element for that.